runtime/vm/JFRChunkWriter.cpp

```diff
@@ -20,6 +20,7 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
  *******************************************************************************/
 #include "JFRUtils.hpp"
+#include "omrformatconsts.h"
```

Comment:
This is not appropriate. The constants are for use with `printf()`, `sprintf()`, etc, not `omrtty_printf()` (via `j9tty_printf()`).

Reply:
It's disappointing to see this still in OMR above the definition of `omrstr_printf()`
```
 * @internal @note Supported, portable format specifiers are described in the document entitled "PortLibrary printf"
 * in the "Inside J9" Lotus Notes database.
```
Here are some sample formats
* `%d` - `int32_t`
* `%lld` - `int64_t`
* `%zd` - `intptr_t`

Have a look at `parseType()` in `omrstr.c` for more details.

Reply:
Yes I took a look at the `omrstr.c` and I think I'm using the correct format specifiers now?